packages/xy-chart/src/utils/chartUtils.js

```diff
@@ -0,0 +1,35 @@
+import { Children } from 'react';
+
+export function callOrValue(maybeFn, ...args) {
+  if (typeof maybeFn === 'function') {
+    return maybeFn(args);
+  }
+  return maybeFn;
+}
```

Comment:
disregard my above `callOrValue` comments... didn't realize you had your own util fn